site/documentation/content/user-guide/device-registry.md

```diff
@@ -138,17 +139,21 @@ The devices in the registry can be managed using the Device Registry Management
 [device related resources]({{< relref "/api/management#devices" >}}).
 
 {{% notice info %}}
+
 The JDBC based registry implementation does not support the following features:
 
-* Registration information can be retrieved using the
-  [search devices]({{< relref "/api/management#devices/searchDevicesForTenant" >}}) operation defined by the Device
-  Registry Management API. The  *filterJson* query parameter currently only allows one filter expression per request,
+* By [search devices]({{< relref "/api/management#devices/searchDevicesForTenant" >}}) operation defined by the Device
+  Registry Management API, the *filterJson* query parameter currently only allows one filter expression per request and
   the *sortJson* query parameter is (currently) being ignored.
   The result set will always be sorted by the device Id in ascending order.
+
+The Mongo DB based registry implementation does not support the following features:
+
+* By [search devices]({{< relref "/api/management#devices/searchDevicesForTenant" >}}) operation defined by the Device
+  Registry Management API, the  *isGateway* query parameter is (currently) being ignored.
```

Comment:
can you please rephrase this to match the section about the JDBC registry?

Reply:
Done